src/main/scala/higherkindness/rules_scala/workers/zinc/compile/FilteredInfos.scala

```diff
@@ -0,0 +1,32 @@
+package sbt.internal.inc
+
+import sbt.internal.inc.SourceInfos
+import xsbti.compile.analysis.SourceInfo
+import xsbti.{Problem, VirtualFileRef}
+
+/**
+ * The consistent analysis format in Zinc does not send the Position in the Problems in the SourceInfos through the read
+ * or write mapper, so the file paths are absolute rather than relativized, which means they are not machine independent
+ * and reproducible.
+ *
+ * This filters out the paths from those Problems/Positions, so we can get a deterministic output.
+ *
+ * TODO: fix this problem in Zinc.
```

Comment:
Is there a bug to reference?

Reply:
Not yet. I also intend to fix this one in the next couple months.